src/renderer/shaders/geometry/point/pointVertexShader.glsl

```diff
@@ -72,6 +69,7 @@ void main(void) {
   if (size == 0. || (stroke.a == 0. && color.a == 0.) || size < orderMinWidth || size >= orderMaxWidth) {
     p.x = 10000.;
   }
+  p.z=0.9999;
```

Comment:
What does this 0.9999 stands for? a very high z? can we add some explaining constant?

Reply:
Actually, I can refactor that logic, hope it helps